scripts/quickstart.sh

```diff
@@ -136,4 +142,35 @@ do
     --cluster.peers             127.0.0.1:19391 &
 done
 
+sleep 0.5
+
+if [ -n "${REMOTE_WRITE_ENABLED}" ]
+then
+  ./thanos remote-write-receive \
```

Comment:
:+1: this is great taht were adding it here as well.

Reply:
either we leave this or remove gossip :slightly_smiling_face:

Reply:
@bwplotka what do you think on this ... personally, I think we should phase out gossip so (in a different PR) change this file to work via the new approach. 
So for this PR I would remove `receive` from this and once the above work has been done add it back in.

Reply:
As we have the e2e tests, that works for me.

Reply:
As there is no git-history of this, I'm going to comment this out for now, but leave it. Does that sound like an ok compromise?

Reply:
I'm happy with that ... can you make sure a comment about why its commented is included :smile:

Reply:
Sounds good to me